components/ui/CompactFeature.vue

```diff
@@ -3,12 +3,12 @@
     <img
       v-if="icon"
       class="compact-feature__icon"
-      :src="`/images/icons/icon${icon}White@3x.png`"
+      :src="`/images/icons/${icon}.svg`"
```

Comment:
Maybe we can add the extension in the parameter in order to generalize this component. I think that also makes a better understanding of how to use the component.